src/tlo/methods/breast_cancer.py

```diff
@@ -759,6 +763,10 @@ def apply(self, person_id, squeeze_factor):
         df.at[person_id, "brc_date_treatment"] = self.sim.date
         df.at[person_id, "brc_stage_at_which_treatment_given"] = df.at[person_id, "brc_status"]
 
+        # Update equipment used with treatment
+        # NB. read only with HSI run and healthsystem.summary logger set at the level INFO or higher
+        self.EQUIPMENT.update({'Anything used for mastectomy as I guess this is about'})
```

Comment:
yes surgery for mastectomy.  Radiotherapy is not at all widely available yet I believe.  Chemotherapy is used but not equipment needed as far as I am aware.